### PR TITLE
Translation to Spanish of the tutorial Part 02

### PR DESCRIPTION
**Pull request by arturomf94**
feat: Translation to Spanish of the Part 02 - Intro to Federated Learning.

This comes as a response to the initiative to translate all the
PySyft tutorials, as it is desirable to learn PySyft in our native
language.

As requested, every part of the tutorial is done via pull request
separately to:

 - Track better the changes in every document.

 - Avoid double work.

Resolves: #2775
See also: #2782

### CODE REVIEW

**Comment by robert-wagner** [comment]
Feel free to add your name as having translated this

**Comment by iamtrask** [comment]
> Feel free to add your name as having translated this

100% agree. Will merge once you add yourself for credit :)

**Comment by arturomf94** [comment]
Thanks @iamtrask and @robert-wagner! I've added myself as an translator. And thank you @ricardopretelt for the heads-up. I've deleted that paragraph. Should be fine now.

File: examples/tutorials/español/Parte 02 - Intro al Aprendizaje Federado.ipynb [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "\n",
     "Autores:\n",
     "- Andrew Trask - Twitter: [@iamtrask](https://twitter.com/iamtrask)\n",
+    "Traductores:\n",
+    "- Arturo Márquez Flores - Twitter [@arturomf94](https://twitter.com/arturomf94)\n",
+    "\n",
     "\n",
     "Traductores:\n",
     "- Arturo Márquez Flores - Twitter: [@arturomf94](https://twitter.com/arturomf94)\n",
```
Comment (ricardopretelt) [inline examples/tutorials/español/Parte 02 - Intro al Aprendizaje Federado.ipynb]: "Autores:\n",
    "- Andrew Trask - Twitter: [@iamtrask](https://twitter.com/iamtrask)\n"


    "Traductores:\n",
    "- Arturo Márquez Flores - Twitter [@arturomf94](https://twitter.com/arturomf94)\n",
    "- Ricardo Pretelt - Twitter: [@ricardopretelt](https://twitter.com/ricardopretelt)"

Comment (arturomf94) [reply examples/tutorials/español/Parte 02 - Intro al Aprendizaje Federado.ipynb]: Hi @ricardopretelt, I think that's  an outdated file. Our names are already in the latest version :)

Comment (ricardopretelt) [reply examples/tutorials/español/Parte 02 - Intro al Aprendizaje Federado.ipynb]: ok, my mistake

**Comment by fdroessler** [comment]
Hi, I have been working on the test suit for all notebooks. I have just updated it but the PR is not merged yet. However, I have now moved all the translations into a subfolder so maybe you could do the same?

As in move the español folder into a subfolder of tutorials:
examples/tutorials/translations/español/

That way once both of our PRs are merged, yours should be automatically included in the test suit.

**Comment by arturomf94** [comment]
> Hi, I have been working on the test suit for all notebooks. I have just updated it but the PR is not merged yet. However, I have now moved all the translations into a subfolder so maybe you could do the same?
> 
> As in move the español folder into a subfolder of tutorials:
> examples/tutorials/translations/español/
> 
> That way once both of our PRs are merged, yours should be automatically included in the test suit.

Done - translations are now in correct subdirectory.

**Comment by ricardopretelt** [comment]
This is ready to merge. The change requested is done.

**Pull request merged**